components/icons/Discord.tsx

```diff
@@ -0,0 +1,16 @@
+export function DiscordIcon({ class: className }: { class?: string }) {
+    return (
+        <svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="2em"
+            height="2em"
```

Comment:
```suggestion
```
**issue:** Generally I try to avoid setting fixed width and height on svgs so that they can be sized in CSS instead.